Kickstarter-iOS/Views/Controllers/PledgeDescriptionViewController.swift

```diff
@@ -196,6 +341,26 @@ private let learnMoreTextViewStyle: TextViewStyle = { (textView: UITextView) ->
   return textView
 }
 
+private let rewardCardViewStyle: ViewStyle = { (view: UIView) -> UIView in
+  view
+    |> \.translatesAutoresizingMaskIntoConstraints .~ false
+}
+
+private let rewardCardContainerMaskViewStyle: ViewStyle = { (view: UIView) -> UIView in
+  view
+    |> roundedStyle(cornerRadius: Styles.grid(1))
+}
+
+private let rewardCardContainerShadowViewStyle: ViewStyle = { (view: UIView) -> UIView in
+  view
+    |> rewardCardShadowStyle
+}
+
+private let rewardCardContainerViewStyle: ViewStyle = { (view: UIView) -> UIView in
+  view
+    |> \.translatesAutoresizingMaskIntoConstraints .~ false
```

Comment:
I've noticed that sometimes it's safer to set `. translatesAutoresizingMaskIntoConstraints .~ false` right in the lazy initializer for the view, so that you're guaranteed that this will be set when you configure & activate your constraints. Otherwise it's a little non-deterministic since this style is only applied in `bindStyles` which is called on `viewWillAppear` and `traitCollectionDidChange`.

Reply:
Sounds good